lib/mixins/property-effects.html

```diff
@@ -2260,6 +2260,15 @@
         if (!wasPreBound) {
           for (let prop in templateInfo.propertyEffects) {
             this._createPropertyAccessor(prop);
+
+            // For any predefined getters without setters, add a read only effect
+            // so that change notifications don't attempt to update the data cache
+            if (this.ACCESSOR_TYPE && this.__dataHasAccessor[prop] === this.ACCESSOR_TYPE.PRE_DEFINED) {
```

Comment:
My thinking was since this was adding a property effect - it shouldn't be in property accessors.  I was trying to consider the cases where PropertyAccessors was mixed in without mixing in PropertyEffects.